DiscordBotsList.Api/Internal/Bot.cs

```diff
@@ -11,53 +11,41 @@ public class Bot : Entity, IDblBot
     {
         internal DiscordBotListApi api;
 
-        [JsonPropertyName("lib")]
-        public string library { get; set; }
+        [JsonPropertyName("lib")] public string library { get; set; }
```

Comment:
Could we not put attributes inline? It'll cause some wonkyness if there's multiple added, and it reads a little weird.